linalg/blockmatrix.cpp

```diff
@@ -320,6 +321,46 @@ void BlockMatrix::EliminateRowCol(Array<int> & ess_bc_dofs, Vector & sol,
    }
 }
 
+void BlockMatrix::EliminateRowCols(const Array<int> & vdofs, BlockMatrix *Ae,
```

Comment:
Could `BlockMatrix &Ae` be used here to avoid potential issue if `Ae == NULL`? I totally understand if there's a reason for `BlockMatrix*` over `BlockMatrix&`... just wanted to suggest the reference if there's no preference.

Reply:
I will put a check to check the pointer